roboteam_ai/src/skills/Receive.h

```diff
@@ -46,15 +46,15 @@ class Receive : public Skill {
     bool readyToReceivePass = false;
 
 
-    bool isInPosition(Vector2 behindTargetPos);
+    bool isInPosition(const Vector2& behindTargetPos = {0, 0});
```

Comment:
What! no! this is bad. the ballplacement target is not always {0,0}. it also mistyfies the behaviour of isInPosition

Reply:
nvm, this had a check